From linux-kernel-owner+greg=40kroah.com-S1751526AbWIFAhF@vger.kernel.org Tue Sep  5 17:38:08 2006
From: Reiner Herrmann <reiner@reiner-h.de>
To: Vojtech Pavlik <vojtech@suse.cz>
Subject: USB: usb/input/usbmouse.c: whitespace cleanup
Date: Wed, 6 Sep 2006 02:37:21 +0200
Cc: linux-kernel@vger.kernel.org, trivial@kernel.org
Content-Disposition: inline
Message-Id: <200609060237.22058.reiner@reiner-h.de>

Replace spaces with tab and change comment indention for better readability.

Signed-off-by: Reiner Herrmann <reiner@reiner-h.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/input/usbmouse.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/input/usbmouse.c
+++ gregkh-2.6/drivers/usb/input/usbmouse.c
@@ -218,7 +218,7 @@ static void usb_mouse_disconnect(struct 
 
 static struct usb_device_id usb_mouse_id_table [] = {
 	{ USB_INTERFACE_INFO(3, 1, 2) },
-    { }						/* Terminating entry */
+	{ }	/* Terminating entry */
 };
 
 MODULE_DEVICE_TABLE (usb, usb_mouse_id_table);
